Update readme to have intelliJ IDEA instructions (#552)
OWASP/wrongsecrets: README.md

We have CycloneDX and OWASP Dependency-check integrated to check dependencies for vulnerabilities.
You can use the OWASP Dependency-checker by calling `mvn dependency-check:aggregate` and `mvn cyclonedx:makeBom` to use CycloneDX to create an SBOM.

### Get the project started in IntelliJ IDEA

Requirements: make sure you have the following tools installed: Docker, Java19, and IntelliJ IDEA.

1. Fork and clone the project as described in the [documentation](https://github.com/OWASP/wrongsecrets/blob/master/CONTRIBUTING.md).

> 2. Import the project in Intellij (e.g. import as mvn project / local sources)

Reviewer: ```suggestion
2. Import the project in IntelliJ (e.g. import as mvn project / local sources)
```

3. Go to the project settings and make sure it uses Java19 (And that the JDK can be found)
4. Go to the IDE settings>Language & Frameworks > Lombok and make sure Lombok processing is enabled

> 5. Open the Maven Tab in your IDEA and run "Reload All Maven Projects" to make the system sync up and downlaod everything.

Reviewer: ```suggestion
5. Open the Maven Tab in your IDEA and run "Reload All Maven Projects" to make the system sync and download everything.
```

> 6. Now run the `main` method in `org.owasp.wrongsecrets.WrongSecretsApplication.java`. This should fail with a stacktrace.

Reviewer: ```suggestion
6. Now run the `main` method in `org.owasp.wrongsecrets.WrongSecretsApplication.java`. This should fail with a stack trace.
```

> 7. Now go to the run configuration of the app and make sure you have the active profile `without-vault` and as environment argument, set `K8S_ENV=docker`.

Reviewer: ```suggestion
7. Now go to the run configuration of the app and make sure you have the active profile `without-vault`. Set `K8S_ENV=docker` as environment argument.
```

> 8. Now repeat step 6: run the app again, now you should have a properly running application.

Reviewer: ```suggestion
8. Repeat step 6: run the app again, you should have a properly running application.
```

> 8. Now repeat step 6: run the app again, now you should have a properly running application.

Reviewer: ```suggestion
8. Repeat step 6: run the app again, you should have a properly running application.
```

> 8. Now repeat step 6: run the app again, now you should have a properly running application.

Reviewer: ```suggestion
8. Repeat step 6: run the app again, you should have a properly running application.
```

> 8. Now repeat step 6: run the app again, now you should have a properly running application.

Reviewer: ```suggestion
8. Repeat step 6: run the app again, you should have a properly running application.
```

> Feel free to start modifying whatever you want to propose as a change, just follow our guidance in the [documentation](https://github.com/OWASP/wrongsecrets/blob/master/CONTRIBUTING.md) to get your work accepted.

Reviewer: ```suggestion
Feel free to edit and propose changes via pull requests. Be sure to follow our guidance in the [documentation](https://github.com/OWASP/wrongsecrets/blob/master/CONTRIBUTING.md) to get your work accepted.
```
